src/app/self-service/users/user.service.ts

```diff
@@ -54,4 +53,14 @@ export class UserService {
     return this.http.disableApiPrefix().get('/assets/mock/user.mock.json');
   }
 
+  /**
+   * Change User Password
+   * @param userId User Id of users
+   * @param password New Password of the user
+   * @returns {Observable<any>}
+   */
+  changePassword(userId: string, passwordObj: any) {
+    return this.http.put(`/users/${userId}`, passwordObj);
```

Comment:
Please follow the documentation for the API details. [Link](https://demo.mifos.io/api-docs/apiLive.htm#selfuser) for the self user service.

Reply:
Resolved